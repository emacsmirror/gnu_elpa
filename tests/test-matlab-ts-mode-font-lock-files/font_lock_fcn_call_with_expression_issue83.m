% -*- matlab-ts -*-

% See: https://github.com/acristoffers/tree-sitter-matlab/issues/83

temp = [1, 2];

myFcn((temp(1) + temp(2)) > 14);

un = find((kmin(:,p,g).') ~= 0);
any(reqext*[-1 0;1 0;0 -1;0 1]>finextpix./data2pixel)

function myFcn(a)
    disp(a)
end
