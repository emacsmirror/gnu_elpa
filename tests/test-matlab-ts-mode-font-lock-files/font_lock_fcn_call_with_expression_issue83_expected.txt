c ccc ccccccccc ccc

c cccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

vvvv o bnD nbD

dddddbbddddbnb o ddddbnbb o nnbD

vv o BBBBbbddddbDDdDdboob oo nbD
BBBbddddddobon nDn nDn onDn nbodddddddddooddddddddddb

kkkkkkkk fffffbvb
    BBBBbdb
kkk
